### ISSUE TITLE
Media unloads/stops when scrolled out of view

### ISSUE THREAD

**Issue by marrus-sh** [tootsuite/mastodon#11160]
### Expected behaviour

Audio should be playable in the background while scrolling oneʼs timeline.

Media elements should not be unloaded when they have sound and are currently playing.

### Actual behaviour

When the media element scrolls out of view, the element is unloaded, and so audio playback stops.

### Steps to reproduce the problem

1. Find an audio post
2. Play the audio
3. Scroll the timeline so that the audio is well outside of the visible area.

The audio playback stops and the playback position is reset, implying that the media element was removed.

### Specifications

Mastodon v2.9.2, tested with the single­‑column layout.

### Other notes

It would be nice if there was a button to quickly scroll to the currently playing media element, in a timeline but that probably depends on #7636.

**Comment by nightpool** [comment]
I think this is maybe not the ideal behavior, but it's better then having an unplaying audio/video element that the user can't find. Twitter also has this behavior, and it's intentional (unlike ours which is a consequence of status element recycling). The ideal behavior, I think, is to "pop out" a media player when the status scrolls out of view, like tumblr does.

(especially bad is the fact that, iirc, media on mastodon loops by default, so if we did this literal thing then you would have to scroll back to stop the media element from playing *every* time you scrolled it out of view)

**Comment by marrus-sh** [comment]
It was more understandable when media was limited to video (having a video playing that you canʼt see is generally undesirable) but if someone publishes a podcast to Masto, it would be very unfortunate if the entire UI became unusable for the duration of the time that the podcast was playing.

(At the very least, if this is deemed WONTFIX, adding a button to the player to open the media in a new tab would be appreciated.)

**Comment by nightpool** [comment]
(you can already open a media attachment in a new tab by right clicking, or long pressing on mobile)

yeah I understand this is bad for long form audio content, but like I said, I think it would hurt the user experience of video and shorter audio posts too severely to implement in the way you describe. I think the Tumblr style solution of the currently playing media element "popping out" into a discrete sidebar position makes the most sense and has the fewest downsides

**Comment by umonaca** [comment]
Please fix this bug because users on our server want the media to be able to play in the background. One helpful note: this bug does not exist in v3.0.1 and it was introduced somewhere in later versions.

**Comment by umonaca** [comment]
By doing git-bisect I can confirm that this bug was introduced in commit 2a5da8c9613b0ef6659edc60831b4fe8e84f3fc8. Before this commit media does not stop playing when scrolled out of view. The pull request #12735 for this commit was not intended to change this behaviour.
Please fix this bug because the users on our server prefer listening to music in the background. Thank you!

**Comment by umonaca** [comment]
Besides, this bug only affects the single column mode, not the advanced layout.
